javascripts/content_scripts/privly.js

```diff
@@ -763,8 +754,6 @@ var privly = {
 
     "use strict";
 
-    return;
```

Comment:
Oh. Well, I thought since ```dispatchResize``` "sends the parent iframe the height of this iframe, only if the "wrapper"  div is not specified", calling ```return;``` in the very beginning would return control already, without reaching ```if``` statements. That's what code climate said as well. I added this after it checks for ```wrapper```. Am I looking at it the wrong way?